examples/tutorial/core.rs

```diff
@@ -1,28 +1,32 @@
 use omnipaxos_core::{
     ballot_leader_election::{BLEConfig, BallotLeaderElection},
     sequence_paxos::{CompactionErr, ReconfigurationRequest, SequencePaxos, SequencePaxosConfig},
-    storage::{memory_storage::MemoryStorage, Snapshot},
+    storage::Snapshot,
     util::LogEntry,
 };
+use omnipaxos_storage::memory::{
+    memory_storage::MemoryStorage, persistent_storage::PersistentState,
+};
 use std::collections::HashMap;
+use zerocopy::{AsBytes, FromBytes};
 
-#[derive(Clone, Debug)]
+#[repr(packed)]
+#[derive(Copy, Clone, Debug, FromBytes, AsBytes)]
 pub struct KeyValue {
-    pub key: String,
```

Comment:
This was a nested problem that came from deriving `FromBytes` and `AsBytes` structs, due to PersistentState requiring it on the KeyValue struct.

I see now that MemoryStorage is used in the core example so i will reverse this, this might be new issue for PersistentStorage i'll have to fix later.